pages/about/policies.html

```diff
@@ -1,444 +1,634 @@
-            <p><b><i>Attendance and Participation Policy</i></b> "Rationale: The policy is necessary to respond to the: Legislative requirements of the Education Welfare Act 2002 which determines that the
-            school must have an attendance policy Role of the National Education Welfare Board Requirements of the Department of Education and Skills..."</p>
-            <div class="ui blue button" onclick="showModal('attendanceparticipation')">
-              Read More <i class="right arrow icon"></i>
+          <div class="ui hidden divider"></div>
+        </div>
+    </div>
+
+    <div class="ui inverted blue vertical container footer segment">
+        <div class="ui center aligned container">
+          <div class="ui section divider"></div>
+            <div class="ui stackable divided grid">
+                <div class="four wide column">
+                    <div class="ui link list">
+                        <a href="https://mountsion.vsware.ie/Login.jsp" target="blank" class="item white">
+                            VSWare
+                        </a>
+                        <a href="mailto:info@mountsioncbssecondary.ie" target="_top" class="item white">
+                            Email
+                        </a>
+                        <a href="https://www.education.ie/en/" target="blank" class="item white">
+                            Department of Education
+                        </a>
+                        <a href="https://www.examinations.ie" target="blank" class="item white">
+                            State Exams Commission
+                        </a>
+                    </div>
+                </div>
+                <div class="four wide column">
+                    <div class="ui link list">
+                        <a href="http://www.pdst.ie" target="blank" class="item white">
+                            P.D.S.T
+                        </a>
+                        <a href="http://www.ncca.ie" target="blank" class="item white">
+                            N.C.C.A
+                        </a>
+                        <a href="http://www.ncca.ie/en/junior-cycle" target="blank" class="item white">
+                            NCCA Junior Cycle
+                        </a>
+                        <a href="http://www.ncca.ie/en/senior-cycle" target="blank" class="item white">
+                            NCCA Senior Cycle
+                        </a>
+                    </div>
+                </div>
+                <div class="eight wide column">
+                    <div class="ui link list">
+                    <h4 class="ui inverted header">Mount Sion C.B.S Waterford</h4>
+                    <p>An all boys school located in the heart of Waterford City</p>
+                    <p>It is the founding school set up by Edmund Rice.</p>
+                    </div>
+                </div>
             </div>
-          </div>
-          <div class="column">
```

Comment:
The antibullyinp policy preview was removed here. I'm not sure that was intended.